lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Player::Human < Player
+
+  def move(board)
+    puts "Please Make Your Move Player #{self.token}: "
+    position = gets
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Please Make Your Move Player #{self.token}: "
  gets
end
```